### PR TITLE
Fix maxZoom

### PR DESCRIPTION
**Pull request by StyXman**
The documentation says:

> and zoomed to the required level of detail using an integer from 0 (for global detail) to 20 (for building level detail)

but the HTML declares a `maxZoom` of 19.

## PR Checklist:
- [ ] All new features have been tested
- [ ] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

**Comment by StyXman** [comment]
Alternatively we could change the docs.

**Comment by StyXman** [comment]
Is it 'normal' that towncrier fails?

**Comment by freakboy3742** [comment]
> Is it 'normal' that towncrier fails?

Yes - if you haven't [provided a change note](https://toga.readthedocs.io/en/stable/how-to/contribute/code.html#add-change-information-for-release-notes)

**Review by freakboy3742 — CHANGES_REQUESTED** [review]
Thanks; In addition to the change note requirement that I flagged in a separate comment, the same change will also need to be made to the Winforms backend, as it also uses the embedded web view approach.

File: changes/mapview-maxZoom-fix.md
Comment (freakboy3742) [inline changes/mapview-maxZoom-fix.md:1]: This filename should be `2557.misc.rst` to placate the town crier check.

**Comment by StyXman** [comment]
I don't quite understand the thing about the towncrier file. The docs say:

> every pull request needs to have a corresponding file in the changes/ directory

It doesn't mention how it should be named or what format.

> You can also see existing examples of news fragments in the changes/ folder

The folder is empty except for the template that collects them all in one file, I guess.

**Comment by freakboy3742** [comment]
> I don't quite understand the thing about the towncrier file. The docs say:
> 
> > every pull request needs to have a corresponding file in the changes/ directory
> 
> It doesn't mention how it should be named or what format.

Those instructions could probably be clearer; it links to the Towncrier docs, which IIRC used to be a lot more concise about what filenames and content to use. As noted in my comment inline - it should be `<ticket_number>.<type>.rst` - so `2557.misc.rst` in this case (since it's a minor change). It could be `2557.bugfix.rst` if you feel it's worth a specific call out in the release notes - my inclination is that it isn't, but I'm willing to be convinced otherwise.

**Comment by StyXman** [comment]
> Those instructions could probably be clearer

I'll try some rephrasing.

**Comment by StyXman** [comment]
Is it normal that

> The folder is empty except for the template that collects them all in one file, I guess.

?

**Comment by StyXman** [comment]
> > Those instructions could probably be clearer
> 
> I'll try some rephrasing.

WIP, will propose another PR:

https://github.com/StyXman/toga/compare/mapview-maxZoom-fix...contribute-code-towncrier

I still need some clarification about the emptiness of the `changes/` directory.

**Comment by freakboy3742** [comment]
Only when you submit a pull request right after we cut a release :-)  

If this PR had been submitted 24 hours ago, that folder would have had a hundred files in it. So, that's an edge case, to be sure.

**Review by freakboy3742 — APPROVED** [review]

**Comment by freakboy3742** [comment]
Thanks for the fix!

**Pull request merged**